src/react-vrpc.js

```diff
@@ -4,17 +4,20 @@ import { VrpcRemote } from 'vrpc'
 const VrpcContext = React.createContext()
```

Comment:
Noting in this file our discussion outside of github. We will merge in this PR but have a new ticket made where we tie the lifecycle of the backend agents, classes and instances all the component lifecycle methods. We are think of using something similar to the react-apollo project